stage/dotnet/dotnet-sdk-enterprise-cloud/GitHub.Octokit.sln

```diff
@@ -0,0 +1,46 @@
+﻿Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31903.59
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{15BC8D73-8583-4A91-9355-970404126E9B}"
+	ProjectSection(SolutionItems) = preProject
+		.editorconfig = .editorconfig
+		.gitignore = .gitignore
+		LICENSE = LICENSE
+		README.md = README.md
+	EndProjectSection
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Tests", "test\Tests.csproj", "{7EF0200D-9F10-4A77-8F73-3E26D3EBD326}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GitHub.Octokit.GHEC.SDK", "src\GitHub.Octokit.GHEC.SDK.csproj", "{242BCEF8-029E-4F6A-863E-40590A9A37BC}"
```

Comment:
I kept the solution name the same here and changed the .csproj to `GitHub.Octokit.GHEC.SDK.csproj`: note the addition of `GHEC` in the middle there. Thoughts?

Reply:
What you're describing was my reasoning exactly. I'm picturing for GHES it looks something like `GitHub.Octokit.GHES.3.19.SDK`.